### PR TITLE
fix JACK support in OpenAL Soft

### PR DESCRIPTION
**Pull request by dlahoti**
The 1.17.0 release of OpenAL Soft has JACK support broken because of a simple typo. This patch fixes it.

see https://github.com/kcat/openal-soft/commit/8ac88304788026370d42449d9cbbd4bb396c01f6

### CODE REVIEW

File: media-libs/openal/openal-1.17.0.ebuild [hunk 1]
```diff
@@ -35,6 +35,12 @@ S=${WORKDIR}/${MY_P}
 
 DOCS="alsoftrc.sample env-vars.txt hrtf.txt ChangeLog README"
 
+src_unpack() {
+	unpack ${A}
+	cd "${S}"
+	epatch "${FILESDIR}/${P}-jackfix.patch"
```
Comment (mgorny) [inline media-libs/openal/openal-1.17.0.ebuild]: In EAPI 5, you apply patches in `src_prepare()`.

**Comment by mgorny** [comment]
@aballier, @gentoochainsaw, @chutz, @radhermit, @yngwin ?

**Comment by dlahoti** [comment]
Is anyone going to merge this? I don't see what the difficulty is.

**Comment by monsieurp** [comment]
Merged.

**Pull request closed without merging**